src/components/BottomSheet.tsx

```diff
@@ -0,0 +1,261 @@
+                                        dayTemp: 21,
+                                        nightTemp: 15,
+                                    },
+                                    {
+                                        date: '2023-03-25 18:00:00.000',
+                                        weatherType: WeatherType.Sunny,
+                                        dayTemp: 21,
+                                        nightTemp: 15,
+                                    },
+                                ]}
+                                extraInfoArray={[
+                                    { type: ExtraInfoType.Precipitation, digitalValue: 12 },
+                                    { type: ExtraInfoType.RealFeel, digitalValue: 12 },
+                                    { type: ExtraInfoType.Visibility, digitalValue: 12 },
+                                    { type: ExtraInfoType.Humidity, digitalValue: 12 },
+                                ]}
+                            />
+                        </BottomSheetScrollView>
+                    </BottomSheet>
+                </GestureHandlerRootView>
+            </View>
+        );
+    }
+    return (
+        <View style={styles.container}>
+            <GestureHandlerRootView style={styles.gestureHandlerContainer}>
+                <BottomSheet
+                    ref={sheetRef}
+                    snapPoints={snapPoints}
+                    index={0}
+                    enableHandlePanningGesture={false}
+                    handleIndicatorStyle={styles.handleIndicator}
+                    onChange={handleSheetChanges}
+                >
+                    <BottomSheetScrollView showsVerticalScrollIndicator={false}>
+                        <View style={styles.buttonContainer}>
+                            <TouchableOpacity style={styles.button} onPress={switchWeather}>
+                                <Text style={[styles.weather, { color: sectionWeather ? colors.BLACK : colors.GRAY }]}>
+                                    {BottomSheetButtons.WEATHER}
+                                </Text>
+                            </TouchableOpacity>
+                            <TouchableOpacity style={styles.button} onPress={switchСlothes}>
+                                <Text style={[styles.clothes, { color: sectionСlothes ? colors.BLACK : colors.GRAY }]}>
+                                    {BottomSheetButtons.CLOTHES}
+                                </Text>
+                            </TouchableOpacity>
+                        </View>
+                    </BottomSheetScrollView>
+                </BottomSheet>
+            </GestureHandlerRootView>
+        </View>
+    );
+};
+
+const styles = StyleSheet.create({
+    gestureHandlerContainer: {
+        flex: 1,
+    },
+    container: {
+        flex: 1,
+        backgroundColor: colors.LIGHT_ORANGE,
+    },
+    buttonContainer: {
+        flexDirection: 'row',
+        columnGap: -75,
```

Comment:
Bad solution, if you use negative values in the columnGap think about what you did wrong